src/main/java/hudson/plugins/accurev/parsers/output/ParseAccuRevVersion.java

```diff
@@ -0,0 +1,29 @@
+package hudson.plugins.accurev.parsers.output;
+
+import hudson.plugins.accurev.AccurevLauncher.ICmdOutputParser;
+import hudson.plugins.accurev.AccurevLauncher.UnhandledAccurevCommandOutput;
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.io.Reader;
+import java.nio.charset.Charset;
+
+public final class ParseAccuRevVersion implements ICmdOutputParser<String, Void> {
```

Comment:
Hmm you already have command called `getAccuRevVersion` however that does not support getting the path to AccuRev

https://github.com/jenkinsci/accurev-plugin/blob/e01b1affe0b3fb648a53b4928907e366306cd025/src/main/java/hudson/plugins/accurev/cmd/GetAccuRevVersion.java

Reply:
Yea.. and in linux it gives issues.. i tried that already.

Reply:
Please replace `GetAccuRevVersion` with the new version :)